### ISSUE TITLE
setProgress(1f) didn't work

### ISSUE THREAD

**Issue by YBill** [airbnb/lottie-android#1248]
##### code show as below：

```
<?xml version="1.0" encoding="utf-8"?>
<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:orientation="vertical"
    tools:context=".LottieActivity">

    <LinearLayout
        android:layout_width="match_parent"
        android:layout_height="wrap_content">

        <Button
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:onClick="handleSetImage"
            android:text="handleSetImage" />

        <Button
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:onClick="handleSetProgress"
            android:text="handleSetProgress" />

    </LinearLayout>

    <com.airbnb.lottie.LottieAnimationView
        android:id="@+id/animation_view"
        android:layout_width="22dp"
        android:layout_height="22dp"
        android:layout_marginLeft="100dp"
        android:layout_marginTop="100dp"
        android:scaleType="centerCrop"
        app:lottie_autoPlay="false"
        app:lottie_fileName="news.json"
        app:lottie_loop="false" />

</LinearLayout>
```
```
public class LottieActivity extends AppCompatActivity {

    private LottieAnimationView mLottie;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_lottie);
        mLottie = findViewById(R.id.animation_view);
    }

    public void handleSetImage(View view) {
        mLottie.setImageResource(R.mipmap.icon_home_news_refresh);
    }

    public void handleSetProgress(View view) {
        mLottie.setAnimation("news.json");
        mLottie.setProgress(1f);

    }

}
```

**Comment by gpeal** [comment]
@YBill Please upgrade to Lottie 3.0.5

**Comment by gpeal** [comment]
If you aren't on androidx, I recommend upgrading, but if you can't there is 3.0.3-support

**Comment by YBill** [comment]
@gpeal Thank you for your answer, My project didn't on androidx, solved my problem with 3.0.3-support.
